worlds/dark_souls_3/__init__.py

```diff
@@ -80,7 +80,7 @@ def create_regions(self):
 
         if self.multiworld.enable_progressive_locations[self.player].value:
             menu_region = self.create_region("Menu", {**progressive_locations, **progressive_locations_2,
-                                                      **progressive_locations_3})
+                                                      **progressive_locations_3, **dlc_progressive_locations})
```

Comment:
its this right here, i realize i only set it so the dlc_progressive_locations turn on with just the enable_progressive_locations, but im currently scratching my head trying to figure out how to write it so those locations get turned on with both enable progressive_locations and enable_dlc

Reply:
i tried a few fixes, tried nesting "enable_dlc" and "enable_progressive_locations" and still got nothing, and even though we didnt add any base game locations, its saying we have 4 extra locations at the moment? just a whole lot of weird on this end atm